gradle.properties

```diff
@@ -22,8 +22,8 @@ kotlinCompilerRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buil
 kotlinVersion=1.3.40-dev-2423
 testKotlinVersion=1.3.40-dev-2423
 # See https://teamcity.jetbrains.com/project.html?projectId=Kotlin_KotlinNativeShared&tab=projectOverview
-sharedRepo=https://dl.bintray.com/jetbrains/kotlin-native-dependencies
-sharedVersion=1.0-dev-57
+sharedRepo=https://teamcity.jetbrains.com/guestAuth/app/rest/builds/buildType:(id:Kotlin_KotlinNativeShared_BuildAndTest),number:1.0-dev-55,branch:clang_for_linux_mingw/artifacts/content/maven
```

Comment:
I'll change when [corresponding PR](https://github.com/JetBrains/kotlin-native-shared/pull/13) will be merged.